test/jdk/java/awt/dnd/DnDFileGroupDescriptor/DnDFileGroupDescriptor.java

```diff
@@ -39,9 +39,7 @@ public void init() {
 
         String[] instructions = {
          "The applet window contains a red field.",
-         "1. Start MS Outlook program. Find and open ",
-         "   the mail form with attachments.",
-         "2. Select attachments from the mail and drag into a red field of applet.",
+         "1. Select any file from File Explorer and drag into a red field of applet.",
```

Comment:
Did you check that "any file from File Explorer" can reproduce the initial JDK-6242241 bug?

Reply:
It will work fine as initial JDK-6242241 bug says "It works fine when dragging from Windows Explorer" but you told to remove Outlook requirement. The JDK-6242241 bug is specifically Outlook issue so that's why I told "regression test was added specially for Microsoft Outlook drag issues"

Reply:
This bug was filed to remove the "Outlook requirement" but it does not mean that we should lose the opportunity to verify the JDK-6242241.

Reply:
I am not sure I am understanding..
JDK-6242241 was Outlook bug so I am not sure how to remove "Outlook requirement" to verify that fix and thus bug? I don't think we can satisfy both requirement!!!
Are you saying we will add @requires but not remove from problem list? 
Or add 
Outlook steps plus
"If Outlook is not present in some platform then you can test with any file from File Explorer"

Reply:
I also think the bug was raised by internal SQE when it was thought that the test could be run on multiple platforms but
http://hg.openjdk.java.net/jdk9/jdk9/jdk/rev/d8efcad28072
made sure it can be run only for windows (although fix not done fully). So, I think we should just add @requires in the html file and not put much stress on removing Outlook requirement.

Reply:
Any more comments on this? I tried removing "Outlook requirement" but couldn't so reinstated back based on the above comments.

Reply:
It will be good to try some possibilities to remove the  "Outlook requirement" and at the same time cover the verification of the JDK-6242241. So yes we need to have both. We probably can create the native app from which the user should be able to drag the data, or we may try to run two java processes, etc to emulate the DnD from the Outlook.